package.json

```diff
@@ -114,13 +113,12 @@
     "react-datepicker": "^1.4.0",
     "react-input-autosize": "^1.1.0",
     "react-select": "^2.4.1",
-    "moment": "^2.18.1"
```

Comment:
I see we're using the same version in SS which is good but I wonder if having both moment-timezone (in SS) and moment (here) is blowing up our bundle or if webpack knows how to do this right?

Reply:
it resolves to the same one (_yarn.lock_ of SS):
```
moment@2.24.0, "moment@>= 2.9.0", moment@^2.18.1, moment@^2.23.0, moment@^2.24.0:
  version "2.24.0"
```

Reply:
great